step-02/js/main.js

```diff
@@ -1,239 +1,278 @@
 'use strict';
 
-var startButton = document.getElementById('startButton');
-var callButton = document.getElementById('callButton');
-var hangupButton = document.getElementById('hangupButton');
-callButton.disabled = true;
-hangupButton.disabled = true;
-startButton.onclick = start;
-callButton.onclick = call;
-hangupButton.onclick = hangup;
-
-var startTime;
-var localVideo = document.getElementById('localVideo');
-var remoteVideo = document.getElementById('remoteVideo');
-
-localVideo.addEventListener('loadedmetadata', function() {
-  trace('Local video videoWidth: ' + this.videoWidth +
-    'px,  videoHeight: ' + this.videoHeight + 'px');
-});
-
-remoteVideo.addEventListener('loadedmetadata', function() {
-  trace('Remote video videoWidth: ' + this.videoWidth +
-    'px,  videoHeight: ' + this.videoHeight + 'px');
-});
-
-remoteVideo.onresize = function() {
-  trace('Remote video size changed to ' +
-    remoteVideo.videoWidth + 'x' + remoteVideo.videoHeight);
-  // We'll use the first onresize callback as an indication that video has started
-  // playing out.
-  if (startTime) {
-    var elapsedTime = window.performance.now() - startTime;
-    trace('Setup time: ' + elapsedTime.toFixed(3) + 'ms');
+// Set up media stream constant and parameters.
+
+// On this codelab, you will be streaming video only (video: true).
+const mediaStreamConstraints = {
+  video: true,
+};
+
+// Set up to exchange only video.
+const offerOptions = {
+  offerToReceiveVideo: 1,
+};
+
+// Defines initial start time of the call (defined as connection between peers).
+let startTime = null;
+
+// Define peer connections, streams and video elements.
+const localVideo = document.getElementById('localVideo');
+const remoteVideo = document.getElementById('remoteVideo');
+
+let localStream;
+let remoteStream;
+
+let localPeerConnection;
+let remotePeerConnection;
+
+
+// Define MediaStreams callbacks.
+
+// Handles success by adding the MediaStream to the video element.
```

Comment:
Nit — use imperative tense: 'Handle success ...' 

I think 'set the MediaStream as the video element src' (or something like that) might be clearer than 'adding'.

Reply:
Google JS Style is on third person. See examples here:
https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

Happy to change them though. Let me know if you prefer to revert.

Reply:
I see what you mean. 

We should follow style as you suggest, i.e. third person declarative.